test/device/intrinsics.jl

```diff
@@ -157,6 +157,11 @@ end
     bufferE = MtlArray(e)
     vecE = Array(SpecialFunctions.erfc.(bufferE))
     @test vecE ≈ SpecialFunctions.erfc.(e)
+
+    f = collect(LinRange(nextfloat(-0.3f0), 0.3f0, 20))
+    bufferF = MtlArray(f)
+    vecF = Array(SpecialFunctions.erfinv.(bufferF))
+    @test vecF ≈ SpecialFunctions.erfinv.(f)
```

Comment:
```suggestion
    @test vecF ≈ SpecialFunctions.erfinv.(f)
    @test all(isnan.(SpecialFunctions.erfinv.(mtl([prevfloat(-1.0f0), nextfloat(1.0f0)]))))
```

And maybe check that outside of the domain returns `NaN`? This is really just a suggestion feel free to not include.

Reply:
I wouldn't test for this since the original function asserts.